### ISSUE TITLE
Create a skin repository

### ISSUE THREAD

**Issue by mc-butler** [MidnightCommander/mc#2147]
> [!IMPORTANT]
> This issue was migrated from Trac:
>
> | **Origin** | https://midnight-commander.org/ticket/2147 |
> | :------------- | :------------------------------------------------------------------------------------------------ |
> | **Reporter** | `buenchaval` |
> | **Mentions** | `gotar@….pl` |

<p>
what about setting a page in the wiki with a skin repository where people can contribute their shining new skins?<br />
</p>
<p>
something like:<br />
</p>
<p>
<a class="ext-link" href="http://www.vi-improved.org/color_sampler_pack/">http://www.vi-improved.org/color_sampler_pack/</a><br />
</p>
<p>
with a screenshot and a link to the ini file, and the possibility to upload new ones.<br />
</p>
<p>
would be nice to have this some day :)<br />
</p>

> [!NOTE]
> Original attachments:
>
> * [skin-mockup-edit.jpg](https://github.com/MidnightCommander/trac-archive/blob/master/attachments/ticket/2147/skin-mockup-edit.jpg) ([raw](https://github.com/MidnightCommander/trac-archive/raw/master/attachments/ticket/2147/skin-mockup-edit.jpg)) by `zaytsev` (@zyv) on <span title="2016-02-23T21:21:24+00:00">`Feb 23, 2016 at 21:21 UTC`</span>

**Comment by zyv** [comment]
https://github.com/MidnightCommander/skins

https://skins.midnight-commander.org

Still need to solve the problem with getting the actual skins from the main `mc` repository. Will be possible after #4684.

**Comment by zyv** [comment]
Finally, everything is live! The skins are taken from this repository when pushing to `master`.
